Source/Support/DispatchSource+Fisticuffs.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  DispatchSource+Fisticuffs.swift
+//  Fisticuffs
+//
+//  Created by Maksym Korytko on 2016-12-22.
+//  Copyright © 2016 theScore. All rights reserved.
+//
+
+import Foundation
+
+extension DispatchSource {
+    /**
+     Creates a single use timer source and resumes it.
+     Once the timer fires, it is automatically invalidated.
+     - parameters:
+        - queue: The dispatch queue that should be used to enque the handler.
+        - interval: Timer interval since now when the handler should be invoked.
+        - leeway: The handler may be called at any time between its scheduled time + leeway.
+        - handler: A function that should be called when the timer fires.
+     
+     - returns: An instance of the DispatchSourceTimer protocol.
+                The returned timer will be in the resumed state.
+     */
+    class func makeScheduledOneshotTimer(
```

Comment:
I chose this name, because there a function in `DispatchSource` called `makeTimerSource`. Now that I'm thinking about it, the function name should be something like `makeScheduledOneshotTimerSource`, since the function returns a `DispatchSourceTimer` object. This is more consistent with the DispatchSource class. Thoughts?